### ISSUE TITLE
Identityref inside augment statement

### ISSUE THREAD

**Issue by danieluribe** [clicon/clixon#77]
Hello,

I am using this yang model: https://github.com/YangModels/yang/blob/master/standard/ieee/draft/802.1/ieee802-dot1q-sched.yang.

It has an augment statement that augments the /if:interfaces/if:interface in ietf-interfaces.

When it comes to the validation of the indentityref, whose base identity is defined in the ieee802-dot1q-sched module it fails. I've been debugging a bit and it results that when it tries to find the base identity it is considered as it was defined in the ietf-interfaces module.

Is this expected?

Daniel Uribe

**Comment by olofhagsand** [comment]
No, I see there are some problems with augmentation, identities and namespaces.
